pinc/metarefresh.inc

```diff
@@ -10,7 +10,7 @@
  * client (determined if headers have been sent or if the output buffer
  * has contents in it).
  */
-function metarefresh($seconds, $url, $title = "", $body = "", $allow_external = false)
+function metarefresh(int $seconds, string $url, string $title = "", string $body = "", bool $allow_external = false)
```

Comment:
Nit: We may want to add a PHP return as this function should always `exit` for security reason.

```suggestion
/** @return never */
function metarefresh(int $seconds, string $url, string $title = "", string $body = "", bool $allow_external = false)
```

Reply:
Great suggestion - thanks!

Reply:
Needed to remove some `exit;`s that were after calls to this function.